src/engine/SimulatorConfig.ts

```diff
@@ -1,7 +1,15 @@
-export default interface SimulatorConfig {
-    world: {
-        xLength: number;
-        zLength: number;
-        includeWalls: boolean;
+import { Vector3d } from "./SimTypes";
+
+export interface SimulatorConfig {
+    defaultWorld: WorldConfig;
+}
+
+export interface WorldConfig {
```

Comment:
This is VERY rough, but my general plan was that the WorldConfig would set up the arena in which the robot will live in. I'm still undecided if the `camera` field should be part of the world config... Maybe I could rename this to `defaultCamera` and it could serve as a starting point for the camera whenever a new world is loaded